src/actions/actions.js

```diff
@@ -725,7 +726,7 @@ module.exports = (router) => {
       });
 
       try {
-        getSettingsForm(action, req, (err, settings) => {
+        getSettingsForm(info, req, (err, settings) => {
```

Comment:
I don't think I understand this change. Won't line 734 fail here?

Reply:
This one is needed for `access` configurations of actions to work. This info is not present on `action` itself, but present on `info` variable.
Also the line 734 won't fail, as `action` is reachable there
![CleanShot 2023-11-09 at 17 36 41](https://github.com/formio/formio/assets/54106407/106221ba-0adf-47a4-b1d8-f2ca7b814004)